mne_icalabel/tests/test_features.py

```diff
@@ -0,0 +1,480 @@
+)
+epo_eeglab_path = str(
+    files("mne_icalabel.tests").joinpath("data/datasets/sample-epo.set")
+)
+
+# ICA activation matrix for raw/epochs
+raw_icaact_eeglab_path = str(
+    files("mne_icalabel.tests").joinpath("data/icaact/icaact-raw.mat")
+)
+epo_icaact_eeglab_path = str(
+    files("mne_icalabel.tests").joinpath("data/icaact/icaact-epo.mat")
+)
+
+# Topography
+raw_topo1_path = str(files("mne_icalabel.tests").joinpath("data/topo/topo1-raw.mat"))
+epo_topo1_path = str(files("mne_icalabel.tests").joinpath("data/topo/topo1-epo.mat"))
+raw_topo_feature_path = str(
+    files("mne_icalabel.tests").joinpath("data/topo/topo-feature-raw.mat")
+)
+epo_topo_feature_path = str(
+    files("mne_icalabel.tests").joinpath("data/topo/topo-feature-epo.mat")
+)
+
+# PSD
+psd_constants_raw_path = str(
+    files("mne_icalabel.tests").joinpath("data/psd/constants-raw.mat")
+)
+psd_psdmed_raw_path = str(
+    files("mne_icalabel.tests").joinpath("data/psd/psdmed-raw.mat")
+)
+psd_steps_raw_path = str(
+    files("mne_icalabel.tests").joinpath("data/psd/psd-step-by-step-raw.mat")
+)
+psd_raw_path = str(files("mne_icalabel.tests").joinpath("data/psd/psd-raw.mat"))
+psd_constants_epo_path = str(
+    files("mne_icalabel.tests").joinpath("data/psd/constants-epo.mat")
+)
+psd_psdmed_epo_path = str(
+    files("mne_icalabel.tests").joinpath("data/psd/psdmed-epo.mat")
+)
+psd_steps_epo_path = str(
+    files("mne_icalabel.tests").joinpath("data/psd/psd-step-by-step-epo.mat")
+)
+psd_epo_path = str(files("mne_icalabel.tests").joinpath("data/psd/psd-epo.mat"))
+
+# Autocorrelations
+autocorr_raw_path = autocorr_short_raw_path = str(
+    files("mne_icalabel.tests").joinpath("data/autocorr/autocorr-raw.mat")
+)
+autocorr_short_raw_path = str(
+    files("mne_icalabel.tests").joinpath("data/autocorr/autocorr-short-raw.mat")
+)
+autocorr_very_short_raw_path = str(
+    files("mne_icalabel.tests").joinpath("data/autocorr/autocorr-very-short-raw.mat")
+)
+autocorr_epo_path = str(
+    files("mne_icalabel.tests").joinpath("data/autocorr/autocorr-epo.mat")
+)
+
+# Complete features
+features_raw_path = str(
+    files("mne_icalabel.tests").joinpath("data/features/features-raw.mat")
+)
+features_epo_path = str(
+    files("mne_icalabel.tests").joinpath("data/features/features-epo.mat")
+)
+
+
+# General readers
+reader = {"raw": read_raw, "epo": read_epochs_eeglab}
+kwargs = {"raw": dict(preload=True), "epo": dict()}
+
+
+# ----------------------------------------------------------------------------
+@pytest.mark.filterwarnings("ignore::RuntimeWarning")
+@pytest.mark.filterwarnings("ignore::FutureWarning")
+@pytest.mark.parametrize(
+    "file, psd_constant_file, eeglab_feature_file",
+    [
+        (raw_eeglab_path, psd_constants_raw_path, features_raw_path),
+        (epo_eeglab_path, psd_constants_epo_path, features_epo_path),
+    ],
+)
+def test_get_features(file, psd_constant_file, eeglab_feature_file):
```

Comment:
```suggestion
def test_get_features_from_precomputed_ica(file, psd_constant_file, eeglab_feature_file):
```

Just to remember what the explicit purpose of the test is. Is this right?